### PR TITLE
Fix git issues again - master did not completely match latest branch

### PR DESCRIPTION
**Pull request by rickboot**
Master was missing a few files and some critical code changes.

### CODE REVIEW

File: client/src/components/Questions/QuestionsAskBar/Button/Button.css [hunk 1]
```diff
@@ -1,12 +1,13 @@
...
   background-color: #ED1923;
-  font: futura;
-  font-size: 14px;
-  font-weight: bold;
   border-style: none;
   padding: 7px 14px;
   width: 149px;
   height: 29px;
   margin: 0;
+  font: futura;
```
Comment (karebae) [inline client/src/components/Questions/QuestionsAskBar/Button/Button.css:9]: WOAH

File: client/src/components/Questions/QuestionsAskBar/QuestionsAskBar.spec.js [hunk 2]
```diff
@@ -23,4 +23,4 @@ describe('QuestionsAskBar unit tests', () => {
     wrapper.find('button').simulate('click')
     expect(mockFn).toHaveBeenCalled();
```
Comment (karebae) [inline client/src/components/Questions/QuestionsAskBar/QuestionsAskBar.spec.js:24]: wow nice!! cool test

**Review by karebae — APPROVED** [review]
looks good!! may the git odds be ever in your favor

File: client/src/components/Questions/QuestionsAskBar/QuestionsAskBar.spec.js (hunk 2 above)
Comment (rickboot) [reply client/src/components/Questions/QuestionsAskBar/QuestionsAskBar.spec.js:24]: that is an old one. it just never appeared on github until now :)

**Pull request merged**